src/domain/entities/Page.ts

```diff
@@ -0,0 +1,11 @@
+export interface Page {
```

Comment:
`Page` is vary not specific name. Maybe `OpenedPage`? and add a well description for it